tests/analysis/indirect/test_losses.py

```diff
@@ -29,16 +30,23 @@ def test_initialize(self):
         )
 
         # 2. Run test.
-
-        _losses = Losses(_config, _analyses)
+        _losses = Losses(
+            _config.graph_files.base_graph_hazard,
+            _analysis,
+            _config.config_data.input_path,
+            _config.config_data.static_path,
+            _config.config_data.output_path,
+            [],
+        )
```

Comment:
you should also be checking the indirect protocol instance:

`assert isinstance(_losses, AnalysisIndirectProtocol)`

This comment is applicable to all the current indirect analysis inits.